<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickRepySettingDialog</class>
 <widget class="QDialog" name="QuickRepySettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border: 1px solid #000000;
   background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #FFFFFF, stop: 0.1 #A3A6A9,  
                                 stop: 0.9 #A3A6A9, stop: 1.0 #FFFFFF);
}
TableView::item::selected
{
	color:white;
	background:blue;  
}</string>
  </property>
  <widget class="QWidget" name="widget_bk" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="TitlebarWidget" name="TitleBar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>590</width>
      <height>370</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>0</y>
       <width>93</width>
       <height>320</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMody">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMoveUp">
        <property name="text">
         <string>上移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMoveDown">
        <property name="text">
         <string>下移</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>580</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbAddGroup">
        <property name="text">
         <string>管理组</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>372</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="text">
         <string>上传配置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="TableView" name="QtableView" native="true">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>0</y>
       <width>371</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
    <widget class="TableView" name="QtableView_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>134</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QWidget</extends>
   <header>Tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
